internal/ignition/ignition.go

```diff
@@ -367,14 +367,14 @@ const secondDayWorkerIgnitionFormat = `{
 
 const tempNMConnectionsDir = "/etc/assisted/network"
 
-var fileNames = [...]string{
-	"bootstrap.ign",
-	masterIgn,
-	"metadata.json",
-	workerIgn,
-	"kubeconfig-noingress",
+var FileNames = [...]string{
+	//"bootstrap.ign",
```

Comment:
if not needed please remove

Reply:
Still WIP so I kept the `hold` label. I'll post a comment when it is ready for review.